pandas/io/stata.py

```diff
@@ -432,7 +432,8 @@ def parse_dates_safe(dates, delta=False, year=False, days=False):
         d = parse_dates_safe(dates, year=True)
         conv_dates = d.year
     else:
-        raise ValueError("fmt %s not understood" % fmt)
```

Comment:
is this where you validate the supported format? (e.g. the tc and such). I think this would be a ValueError if its not in the list (yes unsupported sort of means this, but more likely it was a typo by the user). and datetime w/tz can be not implemented error.